src/server/api/routers/user.ts

```diff
@@ -254,26 +183,20 @@ export const userRouter = createTRPCRouter({
     )
     .query(async ({ ctx, input }) => {
       const limit = input.limit;
-      const users = await ctx.prisma.user.findMany({
-        take: limit + 1,
-        skip: limit * input.page,
-        where: {
-          name: {
-            contains: input.search,
-          },
-        },
+      const dbUsers = await db.query.users.findMany({
+        limit: limit + 1,
+        offset: limit * input.page,
+        where: input.search ? like(users.name, `%${input.search}%`) : undefined,
```

Comment:
Does Drizzle handle sanetization?

Reply:
I'm not completly sure, but I guess

Reply:
Please double check. I want to know this before we merge

Reply:
Okay there definitely using prepared statements:
https://github.com/drizzle-team/drizzle-orm/blob/main/drizzle-orm/src/better-sqlite3/session.ts